src/sqlfluff/core/templaters/base.py

```diff
@@ -159,7 +160,7 @@ def get_line_pos_of_char_pos(
         while nl_idx + 1 < len(ref_str) and ref_str[nl_idx + 1] < char_pos:
             nl_idx += 1
 
-        # NB: +1 because character position is 0-indexed, but the character
+        # NB: +1 because character position is 0-indexed, but the line
```

Comment:
I'm having trouble interpreting this comment. Is it referring _just_ to the `else` block, and if so, is it referring to the first value (`1`) or the second one (`char_pos + 1`)?

Reply:
Yes - and I'll move the comment to make this clearer 👍